### PR TITLE
Add /gif endpoint

### PR DESCRIPTION
**Pull request by andrewgaul**

### CODE REVIEW

**Comment by ahmetalpbalkan** [comment]
Do you personally need this? I don't see this endpoint on httpbin.org. Also this implementation does not provide a the common use case of the GIF (=animated) so I am skeptic about adding this.

**Comment by andrewgaul** [comment]
I can rework this to provide an animated GIF and actually prefer animated.  I need GIFs to test GIF to WebP transcoding in [compy](https://github.com/barnacs/compy).

**Comment by ahmetalpbalkan** [comment]
I'd be okay with a simple animated gif generator (doesn't have to be the same color palette picture). Perhaps just a few frames of a moving rectangle? Not sure if it requires a 3rd party library to do this. (If so it'd be a dealbreaker sort of)

**Comment by andrewgaul** [comment]
Added animation which was less work than I thought.

File: handlers.go [hunk 1]
```diff
@@ -448,6 +450,72 @@ func XMLHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, xmlData)
 }
 
+type Circle struct {
```
Comment (ahmetalpbalkan) [inline handlers.go]: can you please un-export this type?

Comment (andrewgaul) [inline handlers.go]: Done.

**Comment by ahmetalpbalkan** [comment]
Thanks for the pull request!

**Pull request merged**